src/basic/parse-util.h

```diff
@@ -11,6 +11,26 @@
 
 typedef unsigned long loadavg_t;
 
+typedef enum Pcr {
+        PCR_CORE_FIRMWARE      = 0,
+        PCR_PLATFORM_CONFIG    = 1,
+        PCR_EXTERNAL_CODE      = 2,
+        PCR_EXTERNAL_FIRMWARE  = 3,
+        PCR_BOOT_LOADER        = 4,
+        PCR_BOOT_LOADER_CONFIG = 5,
+        PCR_HOST_PLATFORM      = 6,
+        PCR_SECURE_BOOT_POLICY = 7,
+        PCR_KERNEL_INITRD      = 9,
+        PCR_IMA                = 10,
+        PCR_KERNEL_IMAGE_PHASE = 11,
+        PCR_KERNEL_CONFIG      = 12,
+        PCR_SYSEXT_IMAGE       = 13,
+        PCR_SHIM_POLICY        = 14,
+        PCR_SYSTEM_IDENTITY    = 15,
+        _PCR_MAX,
```

Comment:
this should be set to TPM2_PCRS_MAX, because a TPM generally has 24 of these

Reply:
> this should be set to TPM2_PCRS_MAX, because a TPM generally has 24 of these

Alright sir.
Thank you.
If I may ask though when I set up the table pcr_table[_PCR_MAX] will it matter or not that I declare an array of 24 elements and only assign 15?
Totally asking to know, cos as you may have noticed, I'm only a prospective intern, getting a learning opportunity from the very top people in the profession.

Thank you sir.

Reply:
Finally, could I push the fixes a bit later, as I'm currently unable to.
Thank you.